frontend/src/libs/components/meditation-tracks-list/styles.module.scss

```
@@ -0,0 +1,19 @@
+.container {
+  box-sizing: border-box;
+  padding: 251px 36px 0;
+  background-color: var(--white);
+  border-radius: 30px 0 0 30px;
+}
+
+.list {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(400px, 1fr));
+  grid-gap: 20px;
+  margin-top: 0 auto;
+}
+
+@media (width <= 550px) {
```

again, the same problem here
```suggestion
@media (width < 550px) {
```